Library/Formula/bork.rb

```diff
@@ -0,0 +1,14 @@
+class Bork < Formula
+  desc "Bash DSL for config management"
+  homepage "https://github.com/mattly/bork"
+  url "https://github.com/mattly/bork/archive/v0.9.tar.gz"
+  sha256 "3e8217d5fe4de94663dac74af505d05649c9d995998d19d4b247e5d3ecb1dd4b"
+
+  def install
+    prefix.install %w[bin docs lib test types]
+  end
+
+  test do
+    assert_match /bork usage:/, shell_output("#{bin}/bork", 1)
```

Comment:
@bfontaine I had the same question as my first attempt using `system "#{bin}/psshrun"` didn't work. Someone in the IRC channel suggested to solve this like this instead.